README.md

```diff
@@ -170,10 +176,16 @@ VirtualMachinePreference
 centos.8.desktop
 ---
 VirtualMachinePreference
-centos.9
+centos.8.stream
```

Comment:
The centos 8 was EOL on December 31st, 2021 and replaced by centos 8 stream. Do you want to keep it in the list?

Reply:
Ah of course sorry, I'll remove it.